release-notes/docs/3.4.0.md

```diff
@@ -131,3 +131,7 @@ We made several changes to reduce the time it takes to load information on the C
 - [medic#5038](https://github.com/medic/medic/issues/5038): Fix flakey test: Reports Summary Displays correct LHS and RHS summary Concerning reports using patient_id
 - [medic#5099](https://github.com/medic/medic/issues/5099): Fix and re-instate flakey e2e test
 - [medic#5227](https://github.com/medic/medic/issues/5227): Clarify process for running e2e tests locally
+
+## Known Issues
```

Comment:
I think this section should be right at the top, above the "Upgrade notes".

Also... the other headings use sentence case, ie: a lower case "issues"

Reply:
Done